samples/csharp/MicrosoftGraphSDK/Assignment.cs

```diff
@@ -132,112 +137,165 @@ public class Assignment
+            public static async Task<EducationAssignment> DeactivateAsync(
+                GraphServiceClient client,
+                string classId,
+                string assignmentId)
             {
-                throw new GraphException($"PostResourceAsync call: {ex.Message}", ex, classId, assignmentId);
+                try
+                {
+                    return await client.Education
+                        .Classes[classId]
+                        .Assignments[assignmentId]
+                        .Deactivate.PostAsync(requestConfig =>
+                        {
+                            requestConfig.Headers.Add(
+                                "Prefer", "include-unknown-enum-members");
+                        });
+                }
+                catch (Exception ex)
+                {
+                    throw new GraphException($"DeactivateAsync call: {ex.Message}", ex, classId, assignmentId);
+                }
             }
-        }
 
         /// <summary>
-        /// Sets up the assignment resources folder
+        /// Create Assignment Feedback resources folder
+        /// Reference :: https://learn.microsoft.com/en-us/graph/assignments-states-transition
         /// </summary>
         /// <param name="client">Microsoft Graph service client</param>
         /// <param name="classId">User class id</param>
-        /// <param name="assignmentId">Assignment id</param>
+        /// <param name="assignmentId">Assignment id in the class</param>
         /// <returns>EducationAssignment</returns>
-        public static async Task<EducationAssignment> SetupResourcesFolder(
-            GraphServiceClient client,
-            string classId,
-            string assignmentId)
-        {
-            try
+        public static async Task<EducationAssignment> SetUpAssignmentFeedbackResourcesFolder(
+                GraphServiceClient client,
+                string classId,
+                string assignmentId)
             {
-                return await client.Education
-                    .Classes[classId]
-                    .Assignments[assignmentId]
-                    .SetUpResourcesFolder
-                    .PostAsync();
-            }
-            catch (Exception ex)
-            {
-                throw new GraphException($"SetupResourcesFolder call: {ex.Message}", ex, classId, assignmentId);
+                try
+                {
+                    var requestInformation = client.Education.Classes[classId].Assignments[assignmentId].SetUpFeedbackResourcesFolder.ToPostRequestInformation();
+                    var requestBody = new Entity(); // create an empty request body
+                    requestInformation.SetContentFromParsable(client.RequestAdapter, "application/json", requestBody);
+                    var errorMapping = new Dictionary<string, ParsableFactory<IParsable>> {
+                    { "XXX", ODataError.CreateFromDiscriminatorValue }
+                };
```

Comment:
nit: identation and spaces

```suggestion

                    var errorMapping = new Dictionary<string, ParsableFactory<IParsable>> {
                        { "XXX", ODataError.CreateFromDiscriminatorValue }
                    };

```